cdap-docs/admin-manual/source/operations/logging.rst

```diff
@@ -1,77 +1,573 @@
+levels of programs run under Standalone CDAP, you either modify the ``logback.xml`` file,
+or you :ref:`provide a "logback.xml" <application-logback>` with your application
+before it is deployed.
+
+
+.. _logging-monitoring-system-services-logs:
+
+CDAP System Services Logs
+=========================
+As CDAP system services run either on cluster edge nodes or in YARN containers, their
+logging and :ref:`its configuration <logging-monitoring-system-services-logs-configuration>` 
+depends on the service and where it is located.
+
+Retrieving Log Messages from a System Service
+---------------------------------------------
+The log messages emitted by CDAP system services can be retrieved by:
+
+- Using the :ref:`restful-api`: the :ref:`Logging HTTP RESTful API
+  <http-restful-api-logging>` details :ref:`downloading the logs
+  <http-restful-api-logging-downloading-system-logs>` emitted by a system service.
+- Log messages of system services can be viewed in the :ref:`CDAP UI <cdap-ui>`.
+
+- Log messages of a program can be viewed in the :ref:`CDAP UI <cdap-ui>`. In the
+  *Administration* page (accessible through the *CDAP* menu on the far-right)
+  click either a *Logs* icon (such as the *HBase* log icon circled in red, in the 
+  *Component Overview* section):
+
+  .. figure:: /_images/logging/logging-cdap-ui-administration.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+
+      **CDAP UI:** HBase Component, with its *Log* icon circled in red
+
+  You can click on a CDAP system service and a window will appear displaying a log icon
+  (in this case, after clicking *App Fabric*), as shown here circled in red:
+
+  .. figure:: /_images/logging/logging-cdap-ui-administration-app-fabric.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+
+      **CDAP UI:** CDAP *App Fabric* popup window, with its *Log* icon circled in red
+
+System Service Log File Locations
+---------------------------------
+The location of CDAP system service logs depends on the mode of CDAP (Standalone or
+Distributed) and the Hadoop distribution:
+
+- For **Standalone CDAP:** system logs are located in ``<CDAP-SDK-HOME>/logs``.
+
+- For **Distributed CDAP:** system logs are located in ``/var/log/cdap`` (with the
+  exception of Cloudera Manager-based clusters). With Cloudera Manager installations, system
+  log files are located in directories under ``/var/run/cloudera-scm-agent/process``.
+
+.. _logging-monitoring-system-services-logs-configuration:
+
+Configuring System Service Logs
+-------------------------------
+- CDAP system services that run in YARN containers, such as the *Metrics Service*, are
+  configured by the same ``logback-container.xml`` that :ref:`configures user application
+  program logging <configuring-user-application-program-logging>`.
+
+- CDAP system services that run on cluster edge nodes, such as CDAP Master or Router, are
```

Comment:
That sounds right. However, I am not even sure what that list is, and how will we maintain it over time when new services are added? I think you can leave this as is for now. I have created https://issues.cask.co/browse/CDAP-9138 for the improvement.